Packs/DevSecOps/Playbooks/DevSecOps_-_LGTM_-_Analysis_-_SubPB.yml

```
@@ -0,0 +1,1851 @@
+      description: ''
+    nexttasks:
+      '#none#':
+      - "33"
+    scriptarguments:
+      AdditionalPollingCommandArgNames: {}
+      AdditionalPollingCommandArgValues: {}
+      Ids:
+        simple: ${LGTM.code_review_result.id}
+      Interval:
+        simple: "1"
+      PollingCommandArgName:
+        simple: review-id
+      PollingCommandName:
+        simple: lgtm-get-code-review
+      Timeout:
+        simple: "10"
+      dt:
+        simple: LGTM.code_review_result(val.status!=="success").id
+    separatecontext: true
+    loop:
+      iscommand: false
+      exitCondition: ""
+      wait: 1
+      max: 100
+    view: |-
+      {
+        "position": {
+          "x": 510,
+          "y": 160
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+  "33":
+    id: "33"
+    taskid: 7941e853-9bd2-4bed-837e-fd2876c34f97
+    type: regular
+    task:
+      id: 7941e853-9bd2-4bed-837e-fd2876c34f97
+      version: -1
+      name: Refresh LGTM Code Review Details
+      description: Get results of code review
+      script: '|||lgtm-get-code-review'
+      type: regular
+      iscommand: true
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "28"
+    scriptarguments:
+      review-id:
+        simple: ${inputs.LGTM Task ID}
+    separatecontext: false
+    view: |-
+      {
+        "position": {
+          "x": 510,
+          "y": 330
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+view: |-
+  {
+    "linkLabelsPosition": {},
+    "paper": {
+      "dimensions": {
+        "height": 1915,
+        "width": 1280,
+        "x": 50,
+        "y": -340
+      }
+    }
+  }
+inputs:
+- key: Platform
+  value:
+    simple: g
+  required: false
+  description: Git Platform
+  playbookInputQuery:
+- key: Organization
+  value: {}
+  required: false
+  description: Git Project Organization
+  playbookInputQuery:
+- key: Repository
+  value: {}
+  required: false
+  description: Git Repository
+  playbookInputQuery:
+- key: LGTM Task ID
+  value: {}
+  required: false
+  description: LGTM Task ID
+  playbookInputQuery:
+outputs:
+- contextPath: LGTM.project_details.languages.grade
+  description: The grade of the code for this language.
+- contextPath: LGTM.project_details.id
+  description: LGTM Project id
+- contextPath: LGTM.project_details.name
+  description: LGTM Project name
+- contextPath: CreatedIndicator
+  description: commands.local.cmd.new.indicator.output.indicator
+- contextPath: LGTM.code_review_result
+  type: unknown
+- contextPath: LGTM.code_review_result.languages
+  type: unknown
+- contextPath: LGTM.code_review_result.languages.alerts
+  type: unknown
+- contextPath: LGTM.code_review_result.languages.alerts.query.properties
+  type: unknown
+tests:
+- GitLab-test-playbook
+- LGTM-test-playbook
+- playbook-MinIO-Test
+fromversion: 1.0.0
```

```suggestion
fromversion: 5.0.0
```
The version should be consistent across all files in the Pack
(It's currently 5.0.0) Please fix this.